src/app/admin/page.tsx

```diff
@@ -1,13 +1,32 @@
-import styles from '../styles.module.css'
-import test from '@styles/test.module.css'
-import LoginForm from '@/components/LoginForm'
+import styles from '../styles.module.css';
+import test from '@styles/test.module.css';
+import LoginForm from '@/components/LoginForm';
+import Appointments from '@/components/Appointments';
+import { cookies } from 'next/headers';
+import { createServerComponentClient } from '@supabase/auth-helpers-nextjs';
+import AdminLogout from '@/components/AdminLogout';
+
+export default async function Admin() {
+  const supabase = createServerComponentClient({ cookies });
+
+  const {
+    data: { session },
+  } = await supabase.auth.getSession();
 
-export default function Admin() {
   return (
     <main className={styles.main}>
-      <h1 className={test.text}>Welcome to the Admin page!</h1>
-      <p>Please enter your username and password to be granted access.</p>
-      <LoginForm />
+      {!session ? (
+        <>
+          <h1 className={test.text}>Welcome to the Admin page!</h1>
```

Comment:
@mentalcaries You have created `unauthenticated` route but haven't used it. I was wondering that when `session` info is not available, instead of rendering the `LoginForm` component, the user should be redirected to the `unauthenticated` route and that route should display a message and render the `LoginForm` component. What do you think?

Reply:
@5hraddha yea that is a good idea.  But I was kind of wondering what the behaviour should be and if the `unauthenticated` route was necessary in this case.
If they aren't authenticated, they should just be prompted to login. I was thinking they were either logged in, or if not.. they need to login